src/scenic/simulators/robosuite/model.scenic

```diff
@@ -0,0 +1,253 @@
+# src/scenic/simulators/robosuite/model.scenic
+
+"""Scenic world model for RoboSuite - Custom Environments Only."""
+
+from .simulator import RobosuiteSimulator, SetJointPositions, OSCPositionAction
+
+# Global parameters with defaults matching Robosuite's defaults
+param env_config = {}
+param controller_config = None
+param camera_view = None
+param render = True
+param real_time = True
+param speed = 1.0
+param lite_physics = None  # None = use Robosuite default (True)
+
+# Simulator - no more use_environment parameter
+simulator RobosuiteSimulator(
+    render=globalParameters.render,
+    real_time=globalParameters.real_time,
+    speed=globalParameters.speed,
+    env_config=globalParameters.env_config,
+    controller_config=globalParameters.controller_config,
+    camera_view=globalParameters.camera_view,
+    lite_physics=globalParameters.lite_physics
+)
+
+# Default values dictionary
+DEFAULTS = {
+    # Object properties
+    'object_size': 0.03,
+    'density': 1000,
+    'friction': (1.0, 0.005, 0.0001),
+    'solref': (0.02, 1.0),
+    'solimp': (0.9, 0.95, 0.001, 0.5, 2.0),
+    'default_color': (0.5, 0.5, 0.5, 1.0),
+    
+    # Arena properties
+    'table_height': 0.8,
+    'table_width': 0.8,
+    'table_length': 0.8,
+    'table_thickness': 0.05,
+    
+    # Robot properties
+    'robot_width': 0.2,
+    'robot_length': 0.2,
+    'robot_height': 0.5,
+    
+    # Control parameters
+    'control_gain': 3.0,
+    'control_limit': 0.3,
+    'position_tolerance': 0.02,
+    'height_tolerance': 0.02,
+    'gripper_open_steps': 20,
+    'gripper_close_steps': 30,
+    'max_control_steps': 100,
+    'max_lift_steps': 200
+}
+
+# Base classes
+class RoboSuiteObject(Object):
+    """Base class for all RoboSuite objects."""
+    density: DEFAULTS['density']
+    friction: DEFAULTS['friction']
+    solref: DEFAULTS['solref']
+    solimp: DEFAULTS['solimp']
+    shape: BoxShape()
+    allowCollisions: True
```

Comment:
Why allow collisions by default? We don't want to be able to place cubes inside the table, for example.